resources/setups/baseline/module_config.json

```diff
@@ -1,10 +1,13 @@
 {
   "modules": {
     "ipaddr": {
-      "enabled": false,
-      "timeout_sec": 900,
+      "enabled": true,
+      "timeout_sec": 2000,
```

Comment:
why? We can't just arbitrarily increase timeouts because it has a drastic effect on the system overall.  What's the specific case where this is required?

Reply:
@grafnu The multi-subnet functionality in daq/ipaddr.py takes a very long time. This is because the faux device only realises that the subnet range has changed when it comes to renew its DHCP lease (which is 500s). This was the reasoning behind setting the DHCP lease time to 120s compared to 500s.